content/bayes_nn/bayes.md

```diff
@@ -0,0 +1,318 @@
+# Bayesian Neural Network
+
+The usual Neural Network are optimized in way to get fixed value of weights and biases that allows the model perform specific task successfully. Instead in 
+Bayesian Neural Network the weights and biases are the distribution, this type of model could be treated as a ensemble of many neural networks trained by the Bayesian inference.
```

Comment:
"In a Bayesian neural network the weights and biases are distributed rather than fixed. This type of model could be treated as an ensemble of many neural networks, train using a Bayesian inference."